Assets/Scripts/ViconNexusUnityStream/CustomSubjectScript.cs

```diff
@@ -254,60 +254,14 @@ void ProcessData(Data data, string text)
                     {
                         SetPreviousData(marker, _data);
                     }
-                    data.data[marker] = _data;
-                }
-
-                if (gapFillingStrategy == GapFillingStrategy.FillRelative && !dataValid && segment.Value.Count > 1)
```

Comment:
why is this removed?

Reply:
o wait this shouldnt have happened. Maybe this has some changes leftover from something else that i was trying